core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -4741,6 +4743,47 @@ private static class IsNullImplementor extends AbstractRexCallImplementor {
     }
   }
 
+  /** Implementor for the {@code IS NOT DISTINCT FROM} SQL operator. */
+  private static class IsNotDistinctFromImplementor extends AbstractRexCallImplementor {
+    IsNotDistinctFromImplementor() {
+      super("is_not_distinct_from", NullPolicy.NONE, false);
+    }
+
+    @Override public RexToLixTranslator.Result implement(final RexToLixTranslator translator,
+        final RexCall call, final List<RexToLixTranslator.Result> arguments) {
+      final RexToLixTranslator.Result left = arguments.get(0);
```

Comment:
can you add a comment showing the generated java code?

Reply:
I will add comment later.